packages/ckeditor5-engine/docs/framework/guides/deep-dive/custom-element-conversion.md

```diff
@@ -274,6 +270,83 @@ editor.conversion.for( 'dataDowncast' )
+
+```
+<infoBox infoBoxType="Info">    ->    <div class="info-box info-box-info">
+                                          <div class="info-box-title">Info</div>
+                                          <div class="info-box-content">
+    <paragraph>                 ->            <p>
+        Foobar                  ->                Foobar
+    </paragraph>                ->            </p>
+                                          </div>
+</infoBox>                      ->    </div>
+```
+
+Such a mapping can be achieved by registering this callback to the {@link module:engine/conversion/mapper~Mapper#event:modelToViewPosition `Mapper#modelToViewPosition`} event:
+
+```js
+function createModelToViewPositionMapper( view ) {
+    return ( evt, data ) => {
+        const modelPosition = data.modelPosition;
+        const parent = modelPosition.parent;
+
+        // Only mapping of positions that are directly in
+        // the <infoBox> model element should be modified.
+        if ( !parent.is( 'element', 'infoBox' ) ) {
+            return;
+        }
+
+        // Get the mapped view element <div class="info-box">.
+        const viewElement = data.mapper.toViewElement( parent );
+
+        // Find the <div class="info-box-content"> in it.
+        const viewContentElement = findContentViewElement( view, viewElement );
+
+        // Translate the model position offset to the view position offset.
+        data.viewPosition = data.mapper.findPositionIn( viewContentElement, modelPosition.offset );
+    };
+}
+
+// Returns the <div class="info-box-content"> nested in the info box view structure.
+function findContentViewElement( editingView, viewElement ) {
+    for ( const value of editingView.createRangeIn( viewElement ) ) {
+        if ( value.item.is( 'element', 'div' ) && value.item.hasClass( 'info-box-content' ) ) {
+            return value.item;
+        }
+    }
+}
+```
+
+It needs to be plugged into the {@link module:engine/conversion/mapper~Mapper#event:modelToViewPosition `Mapper#modelToViewPosition`} event for both downcast pipelines:
+
+```js
+editor.editing.mapper.on( 'modelToViewPosition', createModelToViewPositionMapper( editor.editing.view ) );
+editor.data.mapper.on( 'modelToViewPosition', createModelToViewPositionMapper( editor.editing.view ) );
+```
+
+<info-box>
+    **Note**: You do not need the reverse position mapping ({@link module:engine/conversion/mapper~Mapper#event:viewToModelPosition from view to model}) because the default view to model position mapping looks for the {@link module:engine/conversion/mapper~Mapper#findMappedViewAncestor mapped view ancestor} and maps offset in respect to that model element.
```

Comment:
```suggestion
    **Note**: You do not need the reverse position mapping ({@link module:engine/conversion/mapper~Mapper#event:viewToModelPosition from the view to the model}) because the default view-to-model position mapping looks for the {@link module:engine/conversion/mapper~Mapper#findMappedViewAncestor mapped view ancestor} and maps the offset in respect to the model element.
```